homeassistant/components/scrape/sensor.py

```diff
@@ -164,6 +166,15 @@ def __init__(
         self._index = index
         self._value_template = value_template
         self._attr_native_value = None
+        if not yaml and trigger_entity_config.get(CONF_UNIQUE_ID):
+            self._attr_name = None
+            self._attr_has_entity_name = True
+            self._attr_device_info = DeviceInfo(
+                entry_type=DeviceEntryType.SERVICE,
+                identifiers={(DOMAIN, trigger_entity_config[CONF_UNIQUE_ID])},
```

Comment:
```suggestion
        if not yaml and (unique_id := trigger_entity_config.get(CONF_UNIQUE_ID)):
            self._attr_name = None
            self._attr_has_entity_name = True
            self._attr_device_info = DeviceInfo(
                entry_type=DeviceEntryType.SERVICE,
                identifiers={(DOMAIN, unique_id)},
```

Personal preference here, but I'd prefer the above and it saves one dict lookup